src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3178,7 +3178,7 @@ void CalcDetailedHcInForDVModel(EnergyPlusData &state,
         // UCSD
         {
             auto const SELECT_CASE_var(state.dataSurface->SurfTAirRef(SurfNum));
```

Comment:
Remove the `SELECT_CASE_var` variable and use `state.dataSurface->SurfTAirRef(SurfNum)` in logic directly.